lib/outline.js

```diff
@@ -1,17 +1,50 @@
-const { PDFDict, PDFName, PDFNumber, PDFString } = require('pdf-lib')
+const { PDFDict, PDFName, PDFNumber, PDFHexString } = require('pdf-lib')
+
+const BuiltInNamedEntities = {
+  'amp': '&',
+  'apos': '\'',
+  'gt': '>',
+  'lt': '<',
+  'nbsp': ' ',
+  'quot': '"',
+}
+const SanitizeXMLRx = /<[^>]+>/g
+const CharRefNamedRx = /&([a-z][a-z]+\d{0,2});/g
+const CharRefDecimalRx = /&#(\d\d\d{0,4});/g
+const CharRefHexRx = /&#x([a-z\d][a-z\d][a-z\d]{0,3});/g
+
+function sanitize (string) {
```

Comment:
I think we can replace this whole function with:

```js
const Entities = require('html-entities').AllHtmlEntities
const entities = new Entities()

function sanitize (string) {
  entities.decode(string)
}
```

Using https://www.npmjs.com/package/html-entities as suggested by @danyill